default/python/common/configure_logging.py

```diff
@@ -78,35 +78,7 @@
 import logging
 import os
 import traceback
-
-try:
-    import freeorion_logger  # FreeOrion logger interface pylint: disable=import-error
-except ImportError as e:
-
-    # Create an alternative logger for use in testing when the server is unavailable
-    class _FreeOrionLoggerForTest(object):
-        """A stub freeorion_logger for testing"""
-        @staticmethod
-        def debug(msg, *args):
-            print(msg)
-
-        @staticmethod
-        def info(msg, *args):
-            print(msg)
-
-        @staticmethod
-        def warn(msg, *args):
-            print(msg)
-
-        @staticmethod
-        def error(msg, *args):
-            print(msg, file=sys.stderr)
-
-        @staticmethod
-        def fatal(msg, *args):
-            print(msg, file=sys.stderr)
-
-    freeorion_logger = _FreeOrionLoggerForTest()
```

Comment:
Reason for deleting this?

Reply:
@Morlic-fo I've catch stack overflow while fixing `freeorion_logger` import. Restored it.